adwords_client/client.py

```diff
@@ -545,9 +545,10 @@ def build_bid_change_operation(internal_operation):
                         utils.cents_as_money(internal_operation.new_bid)
                     )
                 else:
-                    yield operations.add_adgroup_operation(int(internal_operation.campaign_id),
-                                                           int(internal_operation.adgroup_id),
-                                                           utils.cents_as_money(internal_operation.new_bid))
+                    yield operations.add_adgroup_cpc_bid_adjustment_operation(int(internal_operation.campaign_id),
+                                                                              int(internal_operation.adgroup_id),
+                                                                              utils.cents_as_money(
+                                                                                  internal_operation.new_bid))
```

Comment:
What about:
```python
    yield operations.add_adgroup_cpc_bid_adjustment_operation(
        int(internal_operation.campaign_id),
        int(internal_operation.adgroup_id),
        utils.cents_as_money(internal_operation.new_bid),
    )
```
As it is right now, will become bad and worst to read at every param added...